packages/material-ui/src/InputBase/InputBase.js

```diff
@@ -121,6 +124,7 @@ export const styles = theme => {
       height: 'auto',
       resize: 'none',
       padding: 0,
+      lineHeight: '1.1875em',
```

Comment:
Is there a way not to have this style? I'm on my phone, I can't dive too much on it. The fewer custom font style we have, the better customization potential.

Reply:
Well, a few lines above this, the height of input is set to 1.1875em too, so why is this a problem in this case?

Reply:
I would hope the textarea to receive the same styles than the input.